components/ReviewTransactionModal.tsx

```diff
@@ -39,8 +41,14 @@ const ReviewTransactionModal = ({
 }: ReviewTransactionModalProps) => {
   const { txNotify, notifyUser } = useNotifications();
   const { shield, isShielding } = useRailgunTx();
+  const { shieldFees } = useRailgunProvider();
+  const { chain } = useNetwork();
+
   const bigNumberAmount = parseUnits(tokenAmount! || '0', tokenDecimals);
-  const feeAmount = parseUnits(tokenAmount! || '0', tokenDecimals).div('400');
+  const shieldFee = shieldFees[chain?.id || 1];
+  const feeAmount = parseUnits(tokenAmount! || '0', tokenDecimals)
```

Comment:
why do we divide by 10000? is that the unit `shieldFee` is denominated in?

Reply:
Yea, because the units are bips https://github.com/ScopeLift/railgun-shielder/blob/166be6c4270df9b4c85724df3c128667a350c934/hooks/useRailgunProvider.tsx#L8

Reply:
thanks!